cfgov/unprocessed/css/forms.less

```diff
@@ -4,179 +4,79 @@
-    - name: JavaScript setup
-      markup: |
-        <script src="//code.jquery.com/jquery-1.11.0.min.js"></script>
-        <script src="../../static/js/jquery.custom-select.js"></script>
-      notes:
-        - "The jQuery plugin will auto intitiate all select elements with the
-          class of .custom-select and append the the .is-enabled class which
-          activates the styling."
+        - "Wrap the select in .cf-select to help target certain elements"
     tags:
     - cfgov-forms
 */
 
-.custom-select {
-
-    & {
-        .is-enabled& {
-            overflow: hidden;
-            position: relative;
-            min-height: unit((@base-line-height-px + (4px * 2)) / @base-font-size-px, em);
-            // Since an input-group_item can overlap another we need to use z-index
-            // is needed to allow the focused input to pop over unfocused inputs.
-            z-index: 1;
-            border: solid 1px @gray-80;
-            background: @white;
-        }
-    }
+.cf-select {
+    position: relative;
+    display: block;
+    padding: 0;
+    margin-top: 0.5em;
+    border: 1px solid @gray-50;
 
-    &.is-hovered,
-    &.is-focused {
-        z-index: 2;
-        border-color: @pacific;
-        outline: 1px solid @pacific;
-    }
-
-    &_icon {
-        // This element does not get added dynamically so it should be hidden
-        // until JavaScript is enabled.
-        display: none;
-        position: absolute;
-        top: 0;
-        right: 0;
-        z-index: 2;
-        min-width: 1em;
+    select {
+        height: 30px;
+        width: 100%;
         padding: unit( 4px / @base-font-size-px, em )
                  unit( 6px / @base-font-size-px, em );
-        // This will overlay .custom-select_text and make it appear as if any
-        // long words get cut off before they touch the icon.
-        border-left: 4px solid @white;
-        background: @gray-10;
-        color: @gray-50;
-        line-height: @base-line-height;
-        text-align: center;
-        .is-enabled & {
-            display: inline-block;
-        }
-    }
-
-    &_text {
-        position: absolute;
-        top: 0;
-        left: 0;
-        padding: unit( 4px / @base-font-size-px, em )
-                 unit( 10px / @base-font-size-px, em );
-        z-index: 1;
-        font-family: Arial, sans-serif;
-        line-height: @base-line-height;
-        color: @black;
-    }
-
-    &_placeholder {
-        color: @gray;
-    }
-
-    &_select {
-        position: relative;
-        // Push the native select arrow out of view
-        width: 100%;
-        max-height: unit((@base-line-height-px + (4px * 2)) / @base-font-size-px, em);
-        padding: unit( 4px / @base-font-size-px, em );
-        z-index: 3;
-        // Customize the colors
         border: 0;
-        border-radius: 0;
-        background-color: transparent;
-        background-image: none;
+        outline: 2px solid transparent;
```

Comment:
Without the transparent outline, it looks jumpy when outlines are added on hover and focus events.